extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -419,6 +420,29 @@ tags: {  # amp-story-bookend (json)
   }
   attr_lists: "nonce-attr"
 }
+tags: {  # <amp-story-social-share>
+  html_format: AMP
+  tag_name: "AMP-STORY-SOCIAL_SHARE"
+  mandatory_ancestor: "AMP-STORY"
+  mandatory_last_child: true
+  descendant_tag_list: "amp-story-social-share-allowed-descendants"
+  attrs: {
+    name: "layout"
+    mandatory: true
+    value: "nodisplay"
+  }
+  attrs: {
+    name: "src"
+    value_url: {
```

Comment:
By default `allow_relative` is `true`.  Do you want to allow relative URLs or should it only be absolute?

Reply:
Looking at the bookend config (that this will replace when it becomes deprecated), `allow_relative` is a good idea to keep it compatible and there's no drawback from allowing it

Reply:
In that case then the Absolute URL Transformer will need to be updated. Here's the AMP Packager version of that [transformer](https://github.com/ampproject/amppackager/blob/releases/transformer/transformers/absoluteurl.go). Will contact offline with other details.

Reply:
Let me correct myself, we don't need to update it since the attribute is `src` and it makes all `src` attributes absolute. Resolving.